packages/esm-ward-app/src/types/index.ts

```diff
@@ -16,6 +16,11 @@ export interface WardPatientCardProps {
 
 export type WardPatientCardRow = React.FC<WardPatientCardProps>;
 export type WardPatientCardElement = React.FC<WardPatientCardProps>;
+export type WardPatient = {
+  patient: Patient;
+  visit?: Visit;
+  admitted: boolean;
+};
```

Comment:
so I tried to simplify this to singel WardPatient type, instead of having an AdmittedPatient and a InpatientRequest, hopefully this make sense to others.

Note that as we add additional functionality (like the time on ward functionality) we will need to add additional (optional) properties to WardPatient like "latestAdmissionOrTransferEncounter" (which will be a property returned by the inpatient admission endpoint, but not the bed management admissionLocation endpoint).

Reply:
I would have expected this to have an `inpatientAdmission?: InpatientAdmission` property (and also something like a `bedAssignment?: BedAssignment` property)

Reply:
This is following the existing pattern that had been started.   We may not always have an InpatientAdmission (edge case when a patient is in a bed but not admitted).  So we hvae a "Patient" property that we make sure we populate (either from patient/admission or the bed management endpoint) in all cases.

Reply:
I understand the goal of having a common patient property you can rely upon, but that doesn't mean we still shouldn't have an inpatientAdmission property that is available.  That's why I indicated above it should be nullable.  Why wouldn't we include this??